edc-extensions/agreements/retirement-evaluation-store-sql/docs/schema.sql

```diff
@@ -0,0 +1,21 @@
+--
+--  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+--
+--  This program and the accompanying materials are made available under the
+--  terms of the Apache License, Version 2.0 which is available at
+--  https://www.apache.org/licenses/LICENSE-2.0
+--
+--  SPDX-License-Identifier: Apache-2.0
+--
+--  Contributors:
+--        Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+--
+
+
+CREATE TABLE IF NOT EXISTS edc_agreement_retirement
+(
+    contract_agreement_id     VARCHAR      NOT NULL
+        CONSTRAINT agreement_retirement_contract_agreement_id_fk PRIMARY KEY,
```

Comment:
```suggestion
        CONSTRAINT agreement_retirement_contract_agreement_id PRIMARY KEY,
```

Reply:
changed in [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1631/commits/223e47d48cb7fdc2807da6ecb9589541a30d208d).